packages/cpp/ArmoniK.Api.Worker/source/Worker/TaskHandler.cpp

```diff
@@ -52,7 +52,7 @@ void API_WORKER_NAMESPACE::TaskHandler::init() {
   session_id_ = init_request.session_id();
   task_id_ = init_request.task_id();
   task_options_ = init_request.task_options();
-  expected_result_ = init_request.expected_output_keys();
+  expected_result_.assign(init_request.expected_output_keys().begin(), init_request.expected_output_keys().end());
```

Comment:
I would move all the elements instead of copying them. Unfortunately, there is no function to do that:
```suggestion
  expected_result_.clear();
  expected_result_.reserve(init_request.size());
  for (auto& output_key : init_request) {
    expected_result_.push_back(std::move(output_key));
  }
```